ansible/roles/util-pgpass/tasks/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- name: place .pgpass
+  sudo: yes
+  template: src=postgres.pgpass.j2 dest=~/.pgpass mode=0600 backup=yes
```

Comment:
If there is ever a chance that we will want to use this to place a .pgpass in a different user's home directory, then you'll want to remove the `sudo: yes`.

Otherwise, keep the `sudo` and be explicit with the `dest`, making it `/root/.pgpass`. Makes it unambiguous.